typescript/react/portability/i18next/mui-snackbar-message.yaml

```diff
@@ -0,0 +1,21 @@
+rules:
+- id: mui-snackbar-message
+  patterns:
+  - pattern: enqueueSnackbar('$MESSAGE', $X2)
+  - pattern-not: enqueueSnackbar(t($KEY), $X2)
+  message: 'React MUI enqueueSnackbar() title is not internationalized: ''$MESSAGE'''
+  languages:
+  - ts
+  - js
+  severity: WARNING
+  metadata:
+    category: portability
+    technology:
+    - react
+    - mui
+    - i18next
+    source-rule-url: https://www.soluvas.com/
```

Comment:
`source-rule-url` is used for linking to a ported rule - is there a more specific page of soluvas.com describing the specific approach used in this rule?